packages/dx-react-scheduler-material-ui/src/templates/appointment/appointment.test.jsx

```diff
@@ -70,5 +72,35 @@ describe('Appointment', () => {
       expect(clickMock.mock.calls[0][0])
         .toEqual({ target: 'target', data: { text: 'a' } });
     });
+
+    it('should apply clickable class if onClick event exists', () => {
+      const clickMock = jest.fn();
```

Comment:
A mock function isn't required here. Usually `jest.fn()` is useful if we want to check function arguments or call count. 
`onClick={() => {}}` is enough.